### ISSUE TITLE
It can't find info in the Page Assist Github page

### ISSUE THREAD

**Issue by gerroon** [n4ze3m/page-assist#217]
Hi

I just tested to see if it can find the default shortcut for Page Assist in the Github repo. It wasn't able to achieve the task in the web interaction mode. I tested it with Llama3.1 and Gemma latest

open  https://github.com/n4ze3m/page-assist
enable "chat with current page"

```
what is the default keyboard shortcut

llama3-chatqa:latest
Sorry. I cannot find the answer based on the context you provided.
```

**Comment by gerroon** [comment]
Interesting enough 

The Chrome addon finds the answer but the Firefox one can't

**Chrome**
```
what is the default keyboard shortcut page assist



llama3.1:latest
According to the provided context, in Chrome:

Default Keyboard Shortcut: Ctrl+Shift+P

This opens the sidebar of Page Assist.
```


**Firefox**
```

what is the default keyboard shortcut page assist
llama3.1:latest

I don't know. The provided context doesn't mention a default keyboard shortcut for Page Assist. If you're looking for information on how to use or customize Page Assist, I can try to provide more general guidance based on the available documentation.
```

**Comment by n4ze3m** [comment]
Yes, I noticed the issue. It's because of semantic search, which may not provide an accurate answer, but you can turn it off.

https://github.com/user-attachments/assets/6868854e-8092-49cc-a7e3-600edc2c8328

**Comment by gerroon** [comment]
Thanks for the reply. What do we lose by turning it off? Isn't vector embeddings  supposed to be a powerful tool?

**Comment by n4ze3m** [comment]
Yes, it works great most of the time. Page Assist is currently using simple semantic search, which I will improve in the upcoming version :)

**Comment by zono50** [comment]
I can't get this to work with pdfs and when i search a simple term on the website, it can't find it 
![image](https://github.com/user-attachments/assets/6e037c2e-868f-4714-bcc6-91c00d5d4b46)
